exatomic/va/va.py

```diff
@@ -0,0 +1,327 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015-2018, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+"""
+Vibrational Averaging
+#########################
+Collection of classes for VA program
+"""
+import numpy as np
+import pandas as pd
+import csv
+import os
+from exa.util.units import Length
+from exa.util.utility import mkp
+
+_gauss_template='''{link0}
+{route}
+
+{title}
+
+{charge} {mult}
+'''
+float_format = '%    .8f'
+class GenInput:
+    """
+    Supporting class for Vibrational Averaging that will generate input files
+    for a selected program under a certain displacement parameter.
+
+    Computes displaced coordinates for all available normal modes from the equilibrium
+    position by using the displacement vector components contained in the
+    :class:`~exatomic.atom.Frequency` dataframe. It will scale these displacements to a
+    desired type defined by the user with the delta_type keyword. For more information
+    on this keyword see the documentation on the
+    :class:`~exatomic.va.va.GenInputs.gen_delta` function.
+
+    Args:
+        uni (:class:`~exatomic.Universe`): Universe object containg pertinent data
+        delta_type (int): Integer value to define the type of delta parameter to use
+    """
```

Comment:
Very good documentation here. Thanks for being so descriptive!

Reply:
Thanks I really wanted to make sure that people can use it and understand what the code is doing. I'm having that problem in transcribing the perl scripts :)